master/internal/configpolicy/task_config_policy.go

```diff
@@ -24,3 +27,39 @@ type NTSCConfigPolicies struct {
 	InvariantConfig *model.CommandConfig `json:"invariant_config"`
 	Constraints     *model.Constraints   `json:"constraints"`
 }
+
+// PriorityAllowed returns true if the desired priority is within the task config policy limit.
```

Comment:
`// PriorityAllowed returns true if the desired priority is within the limits set by task config policies.`